src/Provider/GeocodeEarth/Tests/IntegrationTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Geocoder package.
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @license    MIT License
+ */
+
+namespace Geocoder\Provider\GeocodeEarth\Tests;
+
+use Geocoder\IntegrationTest\ProviderIntegrationTest;
+use Geocoder\Provider\GeocodeEarth\GeocodeEarth;
+use Http\Client\HttpClient;
+
+/**
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+class IntegrationTest extends ProviderIntegrationTest
+{
+    protected $skippedTests = [
+        'testReverseQueryWithNoResults' => 'We weirdly find stuff here...',
```

Comment:
Could you explain what you call "weird stuff ?

Reply:
Sorry, this was copy pasted from the old mapzen provider.
There's two results here, though the integration test expects none. This is consistent with the old mapzen results for this test (and why it was skipped there as well)

Reply:
Could you copy the results here ?
It's just to have an idea of what kind of results this is.

Reply:
There failure is:
```
Failed asserting that 2 matches expected 0.
Expected :0
Actual   :2
```

Results: https://gist.github.com/atymic/781d61b120f481d087064e59ee9e0459

Reply:
![image](https://user-images.githubusercontent.com/50683531/59484369-d167a980-8eb4-11e9-947f-3e70814059fb.png)

Reply:
We usually don't touch/filter what the API returns.
The goal of this project is to provide easy access to geocoding API, not to decide what should or shouldn't be returned. :)